### PR TITLE
API changelog for release 4.74.0

### PR DESCRIPTION
**Pull request by leslitagordita**

### CODE REVIEW

File: src/content/changelog/4-74-0-2020-09-08.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+title: '4.72.0: 2020-09-08'
+date: 2020-09-08T04:00:00.000Z
+version: 4.74.0
+changelog:
+  - API
+---
+
+### Added
+
+* Added the *Reset Linode Root Password* ([POST /linode/instances/{linodeId}/password](/api/v4/linode-instances-linode-id-password/#post)) endpoint to the *Linode Instances* collection. Use this endpoint to reset the root password for a Linode.
```
Comment (yoshizzle) [inline src/content/changelog/4-74-0-2020-09-08.md:11]: This is specifically for Bare Metal instances. I would check with @Dorthu to see if this should be announced.

File: src/content/changelog/4-74-0-2020-09-08.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+* Added the `bucket_access` field to the *Create an Object Storage Key* ([POST /object-storage/keys](/api/v4/object-storage-keys/#post)) endpoint. This field defines an Object Storage key as a *Limited Access Key*. Limited Access Keys have access to only the bucket(s) that are specified along with `read_write` or `read_only` bucket-level permissions. Limited Access Keys can:
```
Comment (yoshizzle) [inline src/content/changelog/4-74-0-2020-09-08.md:13]: Should check with Vicki to make sure this is correct.

File: src/content/changelog/4-74-0-2020-09-08.md (hunk 1 above)
Comment (Dorthu) [reply src/content/changelog/4-74-0-2020-09-08.md:11]: This endpoint is not specific to bare metal instances

File: src/content/changelog/4-74-0-2020-09-08.md (hunk 2 above)
Comment (victoriaroan) [reply src/content/changelog/4-74-0-2020-09-08.md:13]: Looks good to me! 👍

File: src/content/changelog/4-74-0-2020-09-08.md (hunk 1 above)
Comment (yoshizzle) [reply src/content/changelog/4-74-0-2020-09-08.md:11]: Not the endpoint, the password reset comment itself, which I believe was coming from this line in the changelog: https://bits.linode.com/LinodeAPI/apinext/pull/2910/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR14

**Review by yoshizzle — APPROVED** [review]
Approved based on the outcome of what Smith's says in reply to his comment.

**Pull request merged**